packages/jest-config/src/normalize.js

```diff
@@ -447,11 +487,18 @@ function normalize(options: InitialOptions, argv: Argv) {
     return newOptions;
   }, newOptions);
 
+  newOptions.nonFlagArgs = argv._;
+  newOptions.testPathPattern = buildTestPathPattern(argv);
+  newOptions.json = argv.json;
+  newOptions.lastCommit = argv.lastCommit;
```

Comment:
i'm not sure if it's the right way to do it. Adding these values to the switch block above leaves them `undefined`